### PR TITLE
SP-01

### PR DESCRIPTION
**Pull request by KostiantynVorobiov**

### CODE REVIEW

File: src/main/java/spring/intro/dao/impl/UserDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            session = sessionFactory.openSession();
+            transaction = session.beginTransaction();
+            session.save(user);
+            transaction.commit();
+            logger.info("Added new user " + user.getName() + " successfully");
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't add user " + user.getName() + " to DB ", e);
```
Comment (aleksandr-hrankin) [inline src/main/java/spring/intro/dao/impl/UserDaoImpl.java]: may better display the entire user

Comment (Sofasmile) [inline src/main/java/spring/intro/dao/impl/UserDaoImpl.java]:
```suggestion
            throw new RuntimeException("Can't add user " + user + " to DB ", e);
```

**Review by Sofasmile — APPROVED** [review]

File: src/main/java/spring/intro/dao/impl/UserDaoImpl.java (hunk 1 above)
Comment (KostiantynVorobiov) [reply src/main/java/spring/intro/dao/impl/UserDaoImpl.java]: Done

Comment (KostiantynVorobiov) [reply src/main/java/spring/intro/dao/impl/UserDaoImpl.java]: Done

**Pull request merged**